src/vcpkg/commands.remove.cpp

```diff
@@ -99,66 +99,49 @@ namespace vcpkg::Remove
             spgh.state = InstallState::NOT_INSTALLED;
             write_update(fs, installed, spgh);
 
-            status_db->insert(std::make_unique<StatusParagraph>(std::move(spgh)));
+            status_db.insert(std::make_unique<StatusParagraph>(std::move(spgh)));
         }
     }
 
-    static void print_plan(const std::map<RemovePlanType, std::vector<const RemovePlanAction*>>& group_by_plan_type)
+    constexpr struct AddressOf
```

Comment:
I would prefer this be called `OpAddressOf`/`op_address_of` because it doesn't do what `std::addressof` does.